public/js/core.js

```diff
@@ -59,8 +59,13 @@ tarchive.controller('CoreController', ['$scope', '$filter', 'API', 'NgTableParam
 
 	$scope.askForKey = function() {
 		swal({
-			title: "Welcome to Tarchivepannel!",
-			text: "Enter the key provided by the bot:",
+			 html:true,
+			title:	'Welcome to Tarchivepanel!',
+			text:		'What is Tarchivebot? <br><br> A Telegram bot that archives your chat and provides you with a web interface to look up messages and statistics. <br><br> Add the bot using this link:' +
+							'<a href="https://t.me/Tarchivebot"> Tarchivebot</a><br>' +
```

Comment:
> I'm assuming this Link is clickable without any issues, right?

link tested in chrome/firefox/safari on mac behaviour is as intended 

> I'd probably add the parameter target="_blank".

oops!